### PR TITLE
grunt(core): Remove Excess Grunt Dependencies

### PR DESCRIPTION
**Pull request by codydaig**
@sujeethk Just leftover. I must have missed them in the rebase. Thanks for catching that!

### CODE REVIEW

**Comment by mleanos** [comment]
LGTM

**Comment by codydaig** [comment]
My only other question, is should this line be removed. I don't think it will break anything, but want to be sure. https://github.com/meanjs/mean/blob/8b5466961430627356dd6967b296c8528c044e23/config/assets/default.js#L56

**Comment by sujeethk** [comment]
@codydaig I dont see any reference in the code to that key. But I will leave it up to the contributors to confirm.

Please see below references for Grunt still on the latest master after the other pull was merged.
[Dockerfile#L53](https://github.com/meanjs/mean/blob/master/Dockerfile#L53)
[Dockerfile#L57](https://github.com/meanjs/mean/blob/master/Dockerfile#L57)

[Dockerfile-production#L50](https://github.com/meanjs/mean/blob/master/Dockerfile-production#L50)
[Dockerfile-production#L54](https://github.com/meanjs/mean/blob/master/Dockerfile-production#L54)

[package.json#L113](https://github.com/meanjs/mean/blob/master/package.json#L113)

**Comment by sujeethk** [comment]
@codydaig Looks like the 'gruntConfig' key you mentioned was used only in the gruntfile.js @ L37, L112, L125

File: Dockerfile [hunk 1]
```diff
@@ -50,7 +50,7 @@ RUN sudo apt-get install -yq nodejs \
  && apt-get clean \
  && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
-# Install gem sass for grunt-contrib-sass
+# Install gem sass
 RUN gem install sass
```
Comment (codydaig) [inline Dockerfile]: Is this even needed??

File: Dockerfile-production [hunk 2]
```diff
@@ -47,7 +47,7 @@ RUN sudo apt-get install -yq nodejs \
  && apt-get clean \
  && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
-# Install gem sass for grunt-contrib-sass
+# Install gem sass
 RUN gem install sass
```
Comment (codydaig) [inline Dockerfile-production]: Same here, is this even needed anymore?

Comment (codydaig) [inline Dockerfile-production]: @mleanos It's not Docker. Do users still have to have sass installed through ruby on their computer for it to work?

Comment (mleanos) [inline Dockerfile-production]: I'm just shooting from the hip here, but I *think* that was only required for older version of Node. < 0.10.x?? I haven't looked into that in a while though.

Comment (sujeethk) [inline Dockerfile-production]: I am on Win10 with no Sass/Ruby installed and I just did a simple scss file for core and gulp compiled it to css and i see the style change.

Comment (mleanos) [inline Dockerfile-production]: When I first started using this project I was on Windows 7 & had Node v0.10.x installed; I was required to install ruby/sass with that setup. Now I'm using Windows 10 & Node v6.6.0.

What @sujeethk is saying makes sense, if it's indeed a Node version issue.

@sujeethk Just out of curiosity, what Node version are you running?

Comment (sujeethk) [inline Dockerfile-production]: @mleanos I use NVM to switch versions. I checked on v6.3.1 and v4.5.0 and it works fine. Although when you switch versions you would have to rebuild node-sass using `npm rebuild node-sass`. But everything works fine(including sass compile to css and i see those styles applied) without need for installing ruby or sass.

I am not able to check on v0.12.0 as there is a build error. Honestly we should update node engine to remove support for those old versions.

**Comment by lirantal** [comment]
Hey guys, looks good and indeed all the references from @sujeethk should be removed as well with regards to their grunt reference.

File: Dockerfile-production (hunk 2 above)
Comment (lirantal) [inline Dockerfile-production]: @mleanos @codydaig that gem install isn't required anymore for gulp.
I deprecated it when I introduced the node-sass library which is purely C based and doesn't need the ruby gem dependency

**Comment by codydaig** [comment]
Changes have been made. WIth some LGTM, I will squash on merge.

**Comment by lirantal** [comment]
@codydaig can you also remove these:
https://github.com/meanjs/mean/blob/master/package.json#L115
And in the following remove the grunt-cli global npm lib:
https://github.com/meanjs/mean/blob/master/Dockerfile#L57
https://github.com/meanjs/mean/blob/master/Dockerfile-production#L54

**Comment by codydaig** [comment]
@lirantal Done

**Comment by lirantal** [comment]
Great. Merge whenever you're ready :)

**Pull request merged**